src/js/actions/layers.js

```diff
@@ -821,10 +821,13 @@ define(function (require, exports) {
      * @param {string=} modifier Way of modifying the selection. Possible values
      *  are defined in `adapter/src/lib/layer.js` under `select.vals`.
      *  Default is similar to "select", but the pivot layer will always be cleared.
+     * @param {boolean?} quiet If set, will prevent dispatches as result of this select
+     *  Please use this sparingly, and making sure that you dispatch the correct events
+     *  so UI is updated at some point
      *
      * @returns {Promise}
      */
-    var select = function (document, layerSpec, modifier) {
+    var select = function (document, layerSpec, modifier, quiet) {
```

Comment:
Would now be a good time to convert `modifier` and `quiet` into a single options param? It's a little hard to understand what it means at the call site when you see `this.transfer("layers.select", layers, true, false)`.